packages/framer-motion/src/motion/types.ts

```diff
@@ -73,6 +73,19 @@ export interface CustomStyles {
     image?: string
 }
 
+type CSSPropertiesWithoutTransitionOrSingleTransforms = Omit<
+    CSSProperties,
+    "transition" | "rotate" | "scale" | "perspective"
+>
```

Comment:
One could argue that `perspective` is not really a single transfrom, but I don't know a better name.

Reply:
Removed